website/html/views.py

```diff
@@ -1107,7 +1108,17 @@ def _build_installer(username, platform):
     return False, error_response
 
   try:
-    xmlrpc_proxy = xmlrpclib.ServerProxy(settings.SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC)
+    # Per default this setting is True in DEBUG mode.
+    # Passing an unverified ssl context allows the Custom Installer Builder
+    # to use self-signed server certificates.
+    # c.f. https://github.com/SeattleTestbed/custominstallerbuilder/issues/16
+    if settings.UNVERIFIED_SSL_CONTEXT:
+      xmlrpc_proxy = xmlrpclib.ServerProxy(
+        settings.SEATTLECLEARINGHOUSE_INSTALLER_BUILDER_XMLRPC, 
+        context=ssl._create_unverified_context())

```

Comment:
Isn't this function only available in Python version 2.7+?  What if a user is using version <2.7?  Should there be checks for Python versions in the code, or some kind of disclaimer?

Reply:
Python 2.6 is plainly *unsupported* for running a clearinghouse as of the docs change.